src/Collection.php

```diff
@@ -1517,4 +1517,36 @@ public function stats()
             'collstats' => $this->getName(),
         ));
     }
+
+    /**
+     * Rename collection x to y, with drop previously created target or not option
+     * Note: if dropTarget is false and collection existent, then command will fail
+     *
+     * @param string $target Target name of collection
+     * @param bool $dropTarget Whether to drop previously created collection
+     * @return bool
+     * @throws \Sokil\Mongo\Exception
+     * @throws \MongoException
+     */
+    public function renameCollection($target, $dropTarget = false)
```

Comment:
It is now renaming it's own collection in the db where this collection exists.

Reply:
still dont line bool in armuments. Maybe will be better to `renameCollection` with dropTarget=false, and replaceColection with dropTarget=true?

Then replaceColection will always throw exception if target collection already exists

Reply:
Have you meant: "Then renameCollection will always throw exception if target collection already exists", because with dropTarget=false - mongodb won't try to replace/create new collection if it is present?